_posts/2024-09-27-learning-a-programming-language-with-generative-ai.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: post
+title: Learning a programming language with generative ai
+author: andandersson
+excerpt: >
+  Experiences from learning a programing language with generatve ai.
+tags:
+ - Generative ai
+ - C#
+ - .net
+---
+
+# Background
+
+Generative artificial intelligence, or GenAI, is one of the biggest hypes in the software industry. With the introduction of applications like ChatGPT, AI-algorithms can produce text, generate images and even motion pictures and music. Due to the trends, there have been many discussions how GenAI may affect software development. How can it be utilized,for example for coding, learning new tools and frameworks, code reviewing? 
```

Comment:
With the introduction of applications like ChatGPT, AI-algorithms can produce text, generate images and even motion pictures and music. -> I'd add an comma after "generate images, and ..."

How can it be utilized,for example for coding, learning new tools and frameworks, code reviewing? -> space missing before ",for example"

The last sentence feels a bit confusing. I'd modify at least the structure a bit: How can it be utilized, for example, for coding, learning new tools and frameworks, or code reviewing?

Reply:
corrected